rest/route/distro_routes.go

```diff
@@ -55,3 +57,44 @@ func (dgh *distroGetHandler) Execute(ctx context.Context, sc data.Connector) (Re
 		Result: models,
 	}, nil
 }
+
+type clearTaskQueueHandler struct {
+	distro string
+}
+
+func getClearTaskQueueRouteManager(route string, version int) *RouteManager {
+	return &RouteManager{
+		Route: route,
+		Methods: []MethodHandler{
+			MethodHandler{
+				PrefetchFunctions: []PrefetchFunc{PrefetchUser},
+				Authenticator:     &SuperUserAuthenticator{},
+				RequestHandler:    &clearTaskQueueHandler{},
+				MethodType:        http.MethodDelete,
```

Comment:
This should probably be a post route; delete is usually used to refer to a specific object, such as an individual user.

Reply:
@john-m-liu You missed this comment? any thoughts on this?